### DISCUSSION TITLE
5.x官网中demo运行的G6准确版本是？

### DISCUSSION THREAD

**Discussion by StayAndGo** [antvis/G6#5629] [Q&A] [answered]
问题：Demo在codesandbox中运行不起来（[WASM 布局](https://g6-next.antv.antgroup.com/zh/examples/feature/features/#wasmLayouts)）
![image](https://github.com/antvis/G6/assets/66459813/af7799f9-b017-47af-96ca-964d3eeec168)
看了下代码，在最新的beta.31版本中extension相关的导出方式有修改，估计其他api也有变动；**希望能答复一下新官网中5.0Feature案例使用的依赖版本**，非常感谢🙏。

另外感觉demo中package.json使用固定patch版本如5.0.0-beta.1更稳妥

**Comment by hustcc** [comment, marked as answer]
G6-next 官网中是 G6 [5.x](https://www.npmjs.com/package/@antv/g6)。目前 G6 域名官网是 4.x 的版本。

**Comment by Aarebecca** [comment]
5.x 正式版已经发布，使用 `^5.0.0` 即可
